docs/source/deployment/netlify.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Deploy a fullstack Apollo app with Netlify
+sidebar_title: Netlify
+description: How to run your API using Netlify Functions on AWS Lambda
+---
+
+[Netlify functions](https://www.netlify.com/docs/functions/), is a new way to deploy server-side code to AWS Lambda alongside your frontend, right from GitHub. In this guide, I'll show you why Netlify's functions is a great way to run your whole Apollo app.
+
+1. Use the same platform to deploy your frontend and API code.
+2. Deploy previews of both your frontend and API on every pull request.
+3. No need to worry about scaling containers due to a serverless architecture.
+
+We'll set up a "Hello World" API and frontend for an Apollo app on Netlify. Let's get started!
+
+## Download and Run the Netlify Starter Kit
+
+Download and run the [official Netlify create-react-app and Lambda starter kit](https://github.com/netlify/create-react-app-lambda). This will set us up with a base for our frontend and API code.
+
+
+```bash
+git clone https://github.com/netlify/create-react-app-lambda.git
+cd create-react-app-lambda
+npm install
+```
+
+Let’s take a quick look at the file structure here:
+
+* **package.json:** our dependencies, shared between client and server code. You can also split them up into separate directories later, depending on your preference.
+* **netlify.toml:** configuration for Netlify and the Netlify functions.
+* **src/:** code for our React app.
+* **src/lambda/:** code that will be deployed to Lambda by Netlify.
+
+In `package.json`, you’ll find scripts to run the frontend and lambda code:
+
+```js
+"scripts": {
+  "start": "react-scripts start",
+  "start:lambda": "netlify-lambda serve src/lambda",
+  "build": "react-scripts build",
+  "build:lambda": "netlify-lambda build src/lambda",
+}
+```
+
+Run the code to see that everything is working. We have to open two terminal windows to run the frontend and functions at the same time:
+
+```bash
+# In the first terminal
+npm run start:lambda
```

Comment:
I don't understand what these commands are going to do.  Can we explain that a bit?  Am I actually going to start paying money with `npm run start:lambda`?  Does it actually deploy to Netlify at that point?

Can we provide some details about the difference between `start` and `build`?